app/helpers/application_helper.rb

```diff
@@ -265,6 +247,20 @@ def view_to_url(view, parent = nil)
     end
   end
 
+  def view_to_association(view, parent)
+    case view.db
+    when "OrchestrationStackOutput", "OrchestrationStackParameter", "OrchestrationStackResource"
+                                       then view.db.split(/(?=[A-Z])/).last.downcase.pluralize
+    when 'AdvancedSetting', 'CloudNetwork', 'Filesystem', 'FirewallRule', 'GuestApplication', 'Patch', 'RegistryItem',
+         'ScanHistory'                 then view.db.tableize
+    when "SystemService"
+      case parent
+      when Host then "host_services"
+      when Vm   then @lastaction
+      end
+    end || view.scoped_association
```

Comment:
Yes, I think that's what the `||` is supposed to do but it's a very uncommon syntax if it works this way.